dts/arm/st/g4/stm32g474.dtsi

```diff
@@ -13,7 +13,7 @@
 				compatible = "st,stm32-fdcan";
 				#address-cells = <1>;
 				#size-cells = <0>;
-				reg = <0x40006800 0x400>, <0x4000A800 0x350>;
+				reg = <0x40006800 0x400>, <0x4000A750 0x350>;
```

Comment:
As per Reference Manual, chapter `Memory map and register boundary addresses `
![image](https://user-images.githubusercontent.com/50730894/121903313-775e8080-cd28-11eb-9e4a-23a89abaa25e.png)
So won't it be better to keep start address and enlarge each region?
Meaning for can1:  

> reg = <0x40006400 0x400>, <0x4000A400 **0x400**>;)

and the same for can2     

> reg = <xxx 0x400>, <0x4000A800 **0x400**>;)

and similarly for can3

Reply:
Every instance gets its address from the device-tree
DT_INST_REG_ADDR_BY_NAME(inst, message_ram). The size is defined by the struct size.